data_registry/admin.py

```diff
@@ -132,12 +134,35 @@ def queryset(self, request, queryset):
             return queryset.exclude(qs)
 
 
+class CustomDateFieldListFilter(DateFieldListFilter):
+    def __init__(self, *args, **kwargs):
+        super(CustomDateFieldListFilter, self).__init__(*args, **kwargs)
+
+        now = timezone.now()
```

Comment:
To remember where this is from:

```suggestion
        # https://github.com/django/django/blob/3.2/django/contrib/admin/filters.py#L312-L316
        now = timezone.now()
```